Gemfile

```diff
@@ -7,6 +7,7 @@ gemspec
 
 gem "sanitize", ">= 3.0"
 
+gem 'activesupport'
```

Comment:
RR gem requires activesupport gem. If it is lacked, tests are failed like below.

```
# Running:

EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Finished in 0.005528s, 9044.5353 runs/s, 0.0000 assertions/s.

  1) Error:
TaskListEnableConvertEmptyTaskListOptionTest#test_empty_task_list:
NameError: uninitialized constant ActiveSupport
    /home/hanocha/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rr-3.0.7/lib/rr/integrations/minitest_4.rb:48:in `block (2 levels) in hook'
    /home/hanocha/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rr-3.0.7/lib/rr/integrations/minitest_4.rb:56:in `block (2 levels) in hook'
```